### ISSUE TITLE
`got an unexpected keyword argument 'guess_types'` error

### ISSUE THREAD

**Issue by K4zuki** [ahuigo/xlparser#1]
### Condition
- Ubuntu18.04 (on WSL)
- python 3.6.7
- **`openpyxl == 2.6.2`**
### Problem

`xlparser input.xlsx json.json` fails by `TypeError: load_workbook() got an unexpected keyword argument 'guess_types'`

```
Traceback (most recent call last):
  File "/usr/local/bin/xlparser", line 52, in <module>
    main()
  File "/home/<username>/.local/lib/python3.6/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/home/<username>/.local/lib/python3.6/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/home/<username>/.local/lib/python3.6/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/<username>/.local/lib/python3.6/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/bin/xlparser", line 42, in main
    json.dump(list(rows), open(outfile,'w'), ensure_ascii=False)
  File "/usr/local/lib/python3.6/dist-packages/xlparser.py", line 136, in parseXlsx
    wb = openpyxl.load_workbook(src, read_only=False, guess_types=True)
TypeError: load_workbook() got an unexpected keyword argument 'guess_types'
```

I found similar issue on stackoverflow:
<https://stackoverflow.com/questions/55240409/openpyxl-typeerror-load-workbook-got-an-unexpected-keyword-argument-guess-t>
says `guess_types` has been dropped.

Another hint found here:
<https://bitbucket.org/openpyxl/openpyxl/commits/2408b45757b5a363fd1a5985885e1bc44a25663b?at=default>

So would you please update source to either limit `openpyxl` version or to make it compatible with recent `openpyxl`

**Comment by ahuigo** [comment]
@K4zuki 
I've released a new version of xlparser(version: 0.5.0) just now.
Upgrade you xlparser please.

**Comment by K4zuki** [comment]
@ahuigo I confirmed the new version.
Thanks for your help!
